QUEST.tsv

```diff
@@ -2098,156 +2098,156 @@ QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion t
-QUEST_20150714_002241	Hey…! Hold yourself!
-QUEST_20150714_002242	Thanks. I have too many tasks to do…{nl}I gotta sort out the green cover pages, but I can't find any.{nl}Please look for them from the furnitures inside the monastery.
-QUEST_20150714_002243	For what purpose did you visit this cursed city?{nl}Please go back if you don't have a reason for being here.
-QUEST_20150714_002244	So you are the famous Revelator. {nl}I definitely heard about you before.{nl}
-QUEST_20150714_002245	I will permit your entrance temporarily. {nl}Caliss knightage is looking for competent people like you.
-QUEST_20150714_002246	But, all the authority here belongs to our kingdom military.{nl}I hope you behave well.
-QUEST_20150714_002247	Saltistter? What are you talking about?{nl}You are saying such a non-sense…
-QUEST_20150715_002248	Modestas accepted his defeat and promised his pay back later. When Modestas was about to leave with his horse and wagon, the old Alchemist with a grin took Modesta's attention and made him look at him. {nl}
-QUEST_20150715_002249	Modestas accepted his defeat and promised his pay back later. When Modestas was about to leave with his horse and wagon, the old Alchemist with a grin took Modesta's attention and made him look at him. {nl}
-QUEST_20150717_002250	Please insert the name of your companion.
+QUEST_20150714_002101	$Hones, who is one of them, is lacking in scholarly intelligence. However, I am sure his faithfulness and efforts will help him follow my teachings.
+QUEST_20150714_002102	$The Records of the Hidden Object
+QUEST_20150714_002103	$…I don't know whether or not I should thank the Goddesses for this discovery. My greed for learning can ruin the world.
+QUEST_20150714_002104	$I can't sleep well for fear of my research results being spread by demon investigation. {nl}That's why I wish to pass the rest of the research to the one I can trust.
+QUEST_20150714_002105	$The Inheritance of the Master
+QUEST_20150714_002106	$As the disciples and theologians know, Goddess Bibora is in charge of the library where all records of the world are stored. {nl}I was able to check several records in that fantastic library after the research and long studying I can't tell anyone about at the moment. {nl}
+QUEST_20150714_002107	$From the many records of the Goddess, I was only able to see a few. {nl}Nevertheless, the fact that I checked the existence of the records with the important facts was astonishing. {nl}
+QUEST_20150714_002108	$The record which I can use to find the whereabouts of all the Goddesses… {nl}If you could obtain that secret information, we might be able to predict the Goddesses' future locations by analyzing what they did in the past. {nl}Because of that, your future plans may be disturbed or put at risk. {nl}
+QUEST_20150714_002109	$This is more dangerous because, if the demons find out the truth, their greed towards the library will grow, {nl}and if that greed is fulfilled, the world will be swallowed by evil. That's why…
```

Comment:
I don't like the use of "greed" here, but I can't figure out a more fitting phrasing. Desire maybe?